source/models/defining-models.md

```diff
@@ -79,9 +79,21 @@ export default DS.Model.extend({
 For more about adding computed properties to your classes, see [Computed
 Properties](../../object-model/computed-properties).
 
-If you don't specify the type of the attribute, it will be whatever was
-provided by the server. You can make sure that an attribute is always
-coerced into a particular type by passing a `type` to `attr`:
+Every Model in Ember Data has a special `id` attribute. You do not
+need to manually define this property. This is how Ember Data will be
+able to uniquely identify records when communicating with you
+backend. The `id` attribute can be set for new records however once a
+record has been saved Ember Data will not allow you to update the
+value of the `id` attribute.
+
+
+If you don't specify the type of the attribute, it will be whatever
```

Comment:
Maybe it's more clear if we say that if you don't specify a type, ED does no post processing/transformation. The type isn't as much about type coersion as it is about transforms doing their thing